app/repositories/BCPassengersSessionRepository.scala

```diff
@@ -51,6 +51,8 @@ class BCPassengersSessionRepository @Inject() (
       )
     ) {
 
+  override lazy val requiresTtlIndex = false
```

Comment:
revert this there is broken logic in tests there is nothing wrong with current index